bridges/centralized-ethereum/src/actors/dr_database.rs

```diff
@@ -35,14 +42,38 @@ pub enum DrState {
     Finished,
 }
 
+impl Default for DrState {
```

Comment:
Why is this needed?

Reply:
![image](https://user-images.githubusercontent.com/44392885/115431036-2a1cd180-a205-11eb-84fb-349393b48189.png)

Reply:
Thanks compiler :bow: